### PR TITLE
Template 2: Second half 

### PR DESCRIPTION
**Pull request by camelburrito**
Dropdown , navbar and page specific styles

### CODE REVIEW

File: components/navbar/navbar.css [hunk 1]
```diff
@@ -19,20 +19,16 @@
   background-color: var(--color-primary);
   color: var(--color-font-primary);
   z-index: var(--zindex-highest);
-}
-
-.ampstart-sidebar {
-  background-color: var(--color-primary);
-  color: var(--color-font-primary);
-  width: 80vw;
+  box-shadow: 0px 0px 2px 2px rgba(0,0,0,0.4);
```
Comment (erwinmombay) [inline components/navbar/navbar.css]: remove px if its just 0, also remove the 0 in 0.4

File: components/navbar/navbar.css [hunk 2]
```diff
@@ -19,20 +19,16 @@
...
 }
 
 .ampstart-headerbar-nav .ampstart-nav-item {
   padding: 0 var(--space-2);
-  opacity: 0.9;
+  background: transparent;
+  opacity: 0.8;
```
Comment (erwinmombay) [inline components/navbar/navbar.css]: just .8

Comment (camelburrito) [inline components/navbar/navbar.css]: Done

File: components/navbar/navbar.css (hunk 1 above)
Comment (camelburrito) [inline components/navbar/navbar.css]: Done

**Pull request merged**